src/test/java/org/broadinstitute/consent/http/resources/LivenessResourceTest.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.consent.http.resources;
+
+import org.junit.Test;
+import javax.ws.rs.core.Response;
+import static org.junit.Assert.assertEquals;
+
+public class LivenessResourceTest {
+  @Test
+  public void testHealthy() {
+    LivenessResource resource = new LivenessResource();
+    Response response = resource.healthCheck();
+    assertEquals(200, response.getStatus());
```

Comment:
Nit: Since the endpoint and the api docs describe a "Healthy" string return, you can add an assertion check against `response.getEntity()`